homeassistant/components/apcupsd/sensor.py

```diff
@@ -84,7 +84,7 @@
 
 SPECIFIC_UNITS = {"ITEMP": TEMP_CELSIUS}
 INFERRED_UNITS = {
-    " Minutes": "min",
+    " Minutes": TIME_MINUTES,
     " Seconds": "sec",
```

Comment:
```suggestion
    " Seconds": TIME_SECONDS,
```
(+ the import)

There's also a couple of more sec that should be changed too in travisci/sensor.py and upnp/sensor.py

Reply:
I agree that these should use these new constants as well, but I actually wanted to avoid creating a breaking change with this PR. Considering that this would change the format of the unit of measurement.

Reply:
To me the whole point of changes like this is to make the units used consistent (and help keep them that way). This means changing inconsistent ones to follow the desired unit, otherwise it's pretty much just noise.

In the few similar sweeping changes I've made, I don't think we've flagged them as breaking even though in some cases strictly speaking that's what they are for some scenarios. I don't know why this one would be any different, nor why splitting them to a separate PR would be useful (I certainly hope the plan _is_ to make the consistency changes at some point).

Reply:
Completely agree with you on the importance of consistency. 

You are right it was not marked as a breaking change, so I will change the inconsistent ones as well. Originally I thought that it would be easier to review them separately in contrast to mixing it together into a single PR.

Reply:
Changed in https://github.com/home-assistant/home-assistant/pull/32065/commits/59c869eaff07bee606b3878266b699fcb0578767

Reply:
Seems this was missed, `"sec"` is still there.

Reply:
My bad, changed it now.